conf/utd_sysbio.config

```diff
@@ -0,0 +1,43 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'University of Texas at Dallas HPC cluster profile provided by nf-core/configs'
+  config_profile_contact = 'Edmund Miller(@emiller88)'
+  config_profile_url = 'http://docs.oithpc.utdallas.edu/'
+  singularity_cache_dir = '/scratch/applied-genomics/singularity'
+}
+
+env {
+    TMPDIR = '/home/$USER/scratch/tmp'
+}
+
+singularity {
+  enabled = true
+  envWhitelist='SINGULARITY_BINDPATH'
+  autoMounts = true
+  cacheDir = params.singularity_cache_dir
+}
+
+process {
+  beforeScript = 'module load singularity/3.4.1'
+  executor = 'slurm'
+  queue = { task.memory >= 30.GB && task.cpu <= 16 ? 'normal': 'smallmem' }
+
+
+  withName:STAR_ALIGN {
+    memory = 36.GB
+  }
+
+  withLabel:process_high {
+    cpus = { check_max( 16 * task.attempt, 'cpus' ) }
+    memory = { check_max( 60.GB * task.attempt, 'memory' ) }
+    time = { check_max( 16.h * task.attempt, 'time' ) }
+  }
```

Comment:
This should go into a pipeline specific config file - the labels may not be consistent across all pipelines in regards to what 'e.g.' may refer to.